spark/src/main/java/org/apache/zeppelin/spark/SparkSqlInterpreter.java

```diff
@@ -292,6 +293,7 @@ public Scheduler getScheduler() {
 
   @Override
   public List<String> completion(String buf, int cursor) {
-    return null;
+    SQLContext sqlc = getSparkInterpreter().getSQLContext();
+    return new SparkSqlCompleter(sqlc).completion(buf, cursor);
```

Comment:
isn't this going to create a new instance of SparkSqlCompleter each time code completion is invoked on a text block?